internal/service/vpc/resource_gcp_vpc_peering_connection_test.go

```diff
@@ -0,0 +1,204 @@
+package vpc_test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aiven/aiven-go-client"
+	acc "github.com/aiven/terraform-provider-aiven/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/kelseyhightower/envconfig"
+)
+
+type gcpSecrets struct {
+	GCPProjectID string `envconfig:"GCP_PROJECT_ID" required:"true"`
+	GCPRegion    string `envconfig:"GCP_REGION" required:"true"`
+}
+
+func TestAccAivenGCPPeeringConnection_basic(t *testing.T) {
+	var s gcpSecrets
+
+	err := envconfig.Process("", &s)
+	if err != nil {
+		t.Skipf("Not all values have been provided to establish a GCP VPC peering connection: %s", err)
+	}
+
+	importResourceName := "aiven_gcp_vpc_peering_connection.foo"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { acc.TestAccPreCheck(t) },
+		ProviderFactories: acc.TestAccProviderFactories,
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"google": {
+				Source:            "hashicorp/google",
+				VersionConstraint: ">=4.43.1",
```

Comment:
perhaps this should be changed to be more strict? e.g. strict `4.43.1`

Reply:
I will change to `>=4.0.0,<5.0.0`, we need to have this code working consistently for all the latest versions of GCP operator